html/views.py

```diff
@@ -503,6 +507,114 @@ def download_installers_page(request, build_id):
 
 
 
+def fastlane_page(request):
+  """
+  <Purpose>
+    Renders a key and installer download page for a
+    default user-built seattle build, i.e.:
+      One 80 per-cent vessel, one owner/user
+
+      Note: 
+      owner/user-name can be set in settings.FASTLANE_USER_NAME
+  <Arguments>
+    request:
+      A Django request.
+  <Exceptions>
+    None.
+  <Side Effects>
+    If new session
+    - Creates and stores vesselinfo to appropriate location on disk
+    - Stores generated key pair to session (memory)
+  <Returns>
+    A Django response.
+  """
+
+  try:
+    build_result = False
```

Comment:
The local variables `build_result` (here) and `build_results` (L574) *plus* the `build_results` key in the `request.session` dict (L539) make it kind of hard to grasp what's going on.

I'd vote for naming the first local variable `existing_build_result` or similar; the second could be `new_fastlane_build_results`.